foursight_core/templates/header.html

```diff
@@ -206,9 +206,9 @@
                 </td>
                 <td width="33%" style="padding-right:10pt;white-space:nowrap;color:{% if package == 'foursight' %}#D6F8EA{% else %}#D6EAF8{% endif %};" align="right">
```

Comment:
This kind of conditional would be better handled with a `cgap.css` or `fourfront.css` included to define class-based styles.